docs/tutorial/chat2.md

```diff
@@ -68,4 +68,18 @@ This will bypass the random peer selection process and connect to the specified
 | --- | --- |
 | `/help` | displays available in-chat commands |
 | `/connect` | interactively connect to a new peer |
-| `/nick` | change nickname for current chat session |
\ No newline at end of file
+| `/nick` | change nickname for current chat session |
+
+## `chat2` message protobuf format
+
+Each `chat2` message is encoded as follows
+
+```protobuf
+message Chat2Message {
+  uint64 timestamp = 1;
```

Comment:
This is incorrect, you are using `int64` https://github.com/status-im/nim-waku/blob/b0abbcf5973f085df8802625611471ea56006094/examples/v2/chat2.nim#L63